docs/amphtml/get-started.md

```diff
@@ -11,22 +11,19 @@ localization_priority: Priority
 
 ## Author AMP for Email code
 
-1. Read the [AMP for Email specification](https://amp.dev/documentation/guides-and-tutorials/learn/amp-email-format) to understand the fundamentals. 
-1. If this is the first time you create an AMP email we recommend you to start with this [Tutorial](https://amp.dev/documentation/guides-and-tutorials/start/create_email/?format=email)
-1. Develop your code using [AMP Components](https://amp.dev/documentation/components/?format=email) and test using the [AMP Playground](https://playground.amp.dev/?runtime=amp4email). 
-
+1. Read the [AMP for Email specification](https://amp.dev/documentation/guides-and-tutorials/learn/amp-email-format) to understand the fundamentals.
+1. If this is your first time creating an AMP email we recommend you to start with this [tutorial](https://amp.dev/documentation/guides-and-tutorials/start/create_email/?format=email)
+1. Develop your code using [AMP Components](https://amp.dev/documentation/components/?format=email) and test using the [AMP Playground](https://playground.amp.dev/?runtime=amp4email).
```

Comment:
Could we provide a sample snippet to help them get started?

Reply:
That is already part of the tutorials and playground that we reference on the links.